system_tests/tests/aeroflex2023A.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from parameterized import parameterized
+from utils.ioc_launcher import get_default_ioc_dir
+from common_tests.aeroflex import AeroflexTests, DEVICE_PREFIX, EMULATOR_NAME, TEST_MODES
+
+IOCS = [
+    {
+        'name': DEVICE_PREFIX,
+        'directory': get_default_ioc_dir('aeroflex'),
+        'macros': {'DEV_TYPE': '2023A'},
+        'emulator': EMULATOR_NAME,
+        'lewis_protocol': 'model2023A',
+    },
+]
+
```

Comment:
We do need `TEST_MODES` here though, and for the other device type test file.

Reply:
I define it in `common_tests/areoflex.py` and then import it into the other two test files. If that's an improper way to do it I'm happy to change it.

Reply:
My thinking was, we are running two different suits of tests for two different devices, down the line we might want to have different test modes for each.